File: Server/Server/Models/FlightReservationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Models
{
    public class FlightReservationModel
    {
        public int ReservationId { get; set; }
        public double TotalPrice { get; set; }
        public bool FlightRated { get; set; }
        public bool AirlineRated { get; set; }
        public bool Cancelled { get; set; }
        public CarReservation CarReservation { get; set; } 
        public List<Passenger> Passengers { get; set; }
        public List<int> FlightsIds { get; set; }
    }
}

File: Project/src/app/models/flight-reservation.model.ts
import { CarReservation } from './car-reservation.model';
import { Passenger } from './passenger.model';
import { Flight } from './flight.model';

export class FlightReservation{
   reservationId :number;
   totalPrice :number;
   canRate: boolean;
   flightRated :boolean;
   airlineRated :boolean;
   cancelled :boolean;
   carReservation :any ;
   passengers: Passenger[];
   flightPrice:number[]=[];
   flights:Flight[];
   flightsIds:number[]=[];

  constructor(){
    this.passengers =[];
    this.cancelled=false;
    this.airlineRated=false;
    this.flightRated=false;
    this.flights = [];
  }
}
